internal/wizard/project/create.go

```diff
@@ -0,0 +1,291 @@
+					Validate(func(selectedOrg string) error {
+						for _, org := range organizations {
+							if org.Name == selectedOrg {
+								orgProjectCount := len(org.Projects)
+								if orgProjectCount >= org.QuotaProject && org.QuotaProject >= 0 {
+									orgValidationErr = true
+									return nil
+								}
+							}
+						}
+						return nil
+					}),
+			),
+		).WithTheme(themes.ThemeLagoon())
+
+		err = form2.Run()
+		if err != nil {
+			return nil, err
+		}
+
+		if orgValidationErr {
+			var orgRetryResp string
+			orgRetryRespForm := huh.NewForm(
+				huh.NewGroup(
+					huh.NewSelect[string]().
+						Title("Organization quota exceeded. What would you like to do?").
+						Options(
+							huh.NewOption("Go back to previous form", "back"),
+							huh.NewOption("Cancel wizard", "cancel"),
+						).
+						Value(&orgRetryResp),
+				),
+			).WithTheme(themes.ThemeLagoon())
+
+			err := orgRetryRespForm.Run()
+			if err != nil {
+				return nil, err
+			}
+
+			switch orgRetryResp {
+			case "back":
+				return RunCreateWizard(lc)
+			case "cancel":
+				return nil, huh.ErrUserAborted
+			}
+		}
+	}
+
+	var deployTargets *[]schema.DeployTarget
+	var err error
+	if organizationConfirm {
+		orgDeploytargets, err := util.GetOrgDeployTargets(lc, config.OrganizationDetails.Name)
+		if err != nil {
+			return nil, err
+		}
+		deployTargets = &orgDeploytargets
+	} else {
+		deployTargets, err = lagoon.ListDeployTargets(context.TODO(), lc)
+		if err != nil {
+			return nil, err
+		}
+		if deployTargets == nil {
+			return nil, errors.New("no deployTargets found for user")
+		}
+	}
+	var additionalFields bool
+	options := make([]huh.Option[uint], len(*deployTargets))
+	for i, target := range *deployTargets {
+		options[i] = huh.NewOption(target.Name, target.ID)
+	}
+	form3 := huh.NewForm(
+		huh.NewGroup(
+			huh.NewInput().
+				Title("Enter the Git URL for your project").
+				Value(&config.Input.GitURL).
+				Validate(func(str string) error {
+					if !util.IsValidGitURL(config.Input.GitURL) {
+						return errors.New("invalid Git URL")
+					}
+					return nil
+				}),
+			huh.NewInput().
+				Title("Enter the branch name for the production environment").
+				Value(&config.Input.ProductionEnvironment),
+			huh.NewSelect[uint]().
+				Title("Select a deploy target").
+				Options(options...).
+				Value(&config.Input.Openshift),
+			huh.NewConfirm().
+				Title("Do you want to define any other fields?").
+				Value(&additionalFields),
+		),
+	).WithTheme(themes.ThemeLagoon())
+
+	err = form3.Run()
+	if err != nil {
+		return nil, err
+	}
+
+	if additionalFields {
+		var fields []string
+		additionalFieldsOptions := []huh.Option[string]{
+			huh.NewOption("branches: Which branches should be deployed", "Branches"),
+			huh.NewOption("pullrequests: Which Pull Requests should be deployed", "PullRequests"),
+		}
+		if organizationConfirm {
+			additionalFieldsOptions = append(additionalFieldsOptions, huh.NewOption("owner", "AddOrgOwner"))
```

Comment:
This may need to be this?
```
owner: Add your account to the project group during creation
```
<img width="494" height="92" alt="image" src="https://github.com/user-attachments/assets/98fc8ddf-b328-416e-8127-ea4df38adde1" />